src/resolutions/lesson-02/1-intro.ts

```diff
@@ -0,0 +1,22 @@
+import { Equal, Expect } from "../../../helper";
+
+/*1. apply the knowledge from lesson 01 to type this funciton correctly,
+	notice the test is also asking us to type it as Set<unknown> when no generic is passed 
+    to the function. 
+*/
+
+type SetType = string | number | unknown;
+
+export const createOwnSet = <T extends SetType = unknown>(t: Iterable<T> = []) => {
+  return new Set(t);
+};
+
```

Comment:
no es necesario asignar a nuestro argumento el tipo Iterable<T> = [], ya que solo necesitamos pasarle un string o numero, un iterable es distinto. ademas por la sintaxys que es igual a un arrat [] confunde a los que solo han trabajado con JS.
ejemplo: esto ["set"] es un array de string, no un string para JS.

Reply:
Thx, addressed in next commit.